### ISSUE TITLE
wb_load fails

### ISSUE THREAD

**Issue by sigbertklinke** [JanMarvin/openxlsx2#261]
Hi,

the following MWE fails

```
library("magrittr")
library("openxlsx2")
who <- c("Emma", "Charlotte", "Amelia", "Ava", "Sophia")
for (i in who) {
#  browser()
  file  <- paste0(substr(i, 1, 1), ".xlsx")
  message(file)
  wb    <- if (!file.exists(file)) wb_load("Vorlage0.xlsx") else wb_load(file)
  wb$clone_worksheet("Vorlage", i)
  wb$add_data(x = i, startCol = 2, startRow = 1)
  wb$save(file)
}
```

with a error message when calling `wb_load(file)`

```
E.xlsx
C.xlsx
A.xlsx
A.xlsx
Fehler in names(z) <- as.character(nam) : 
  Attribut 'names' [3] muss dieselbe Länge haben wie der Vektor [1]
```

[Vorlage0.xlsx](https://github.com/JanMarvin/openxlsx2/files/9048638/Vorlage0.xlsx)

**Comment by JanMarvin** [comment]
Hi @sigbertklinke , confirmed the bug. It's caused by the headerfooter logic. Initially we read the file "Vorlage0.xlsx", when saving it and trying to read it again, we fail:

```R
nam
[1] "L" "C" "R"
z
[1] "&amp;&quot;Times New Roman,Standard&quot;&amp;12&amp;A"
```

**Comment by JanMarvin** [comment]
Pushed a fix to #262 Previously exported files are indeed broken, thanks for the report! Your file has only a center entry in header and footer (visible in print preview in LibreOffice). Our export for this type of files was broken.

**Comment by JanMarvin** [comment]
While we could offer an option to skip the imports of header and footer to read our own broken files, we are still in development, therefore I decided that I don't care to much.
